inttests/storagepool_test.go

```diff
@@ -223,3 +223,13 @@ func TestCreateDeleteStoragePool(t *testing.T) {
 	assert.NotNil(t, err)
 
 }
+
+// TestGetSDSStoragePool gets the SDS instances associated with storage pool
+func TestGetSDSStoragePool(t *testing.T) {
```

Comment:
Can you also add the negative test scenario where error is notnil

Reply:
@VamsiSiddu-7 As discussed over chat, no parameter is getting passed as the storage pool is returned by the getStoragePool(t) function. I am not sure if we can add the negative test.

I have run the tests. Please find the below screenshot for the same.

![image](https://user-images.githubusercontent.com/60608990/207289809-2ba18405-eede-4eb5-9c48-bfc01763da1a.png)